server/routes/requests.js

```diff
@@ -11,6 +12,22 @@ router.get('/', async (req, res, next) => {
     })
 })
 
+router.patch('/applyRequesterTimezone', async (req, res, next) => {
```

Comment:
This isn't necessary if we change how /alumni/all/:timezone works BUT it probably makes more sense to do it this way